docs/Upgrade.md

```diff
@@ -2636,6 +2636,21 @@ import { BooleanInput } from 'react-admin';
 />
 ```
 
+### `BooleanInput` `options` Prop Has Been Removed
```

Comment:
It's a bit too late to introduce breaking change. So either we wait for 5.0, or we do it in a backwards compatible way.